.travis.yml

```diff
@@ -32,21 +32,21 @@ jobs:
       python: "2.7"
       env: TOXENV=py27-sqlite
       install: pip install codecov tox
-      script: tox -- --exitfirst -m functional -k 'not harvesting'
+      script: tox -- --exitfirst -m functional -k 'not harvesting' --functional-prefer-diffs
```

Comment:
Didn't we agree that the `--functional-prefer-diffs` flag was not to become the default behaviour for tests? If so I don't think it should be set on Travis either, since these are our default CI tests.

Reply:
@ricardogsilva agree/+1.  I turned these on temporarily to try to see what the error was (my local built fine).